### ISSUE TITLE
Bar for Ł slightly unbalanced

### ISSUE THREAD

**Issue by katef** [skosch/Crimson#24]
It strikes me that the bar for Ł seems a little bit too long on the right hand side. I find it unbalanced. I don't think it should be exactly symmetrical, but I do think it should be a *little* bit shorter on the right hand side. Looking at other similar fonts, that seems to be the consensus. What do people think?

Unrelated, there's also the optical illusion that a diagonal going through a vertical stem will have the right hand half appearing higher than it actually is. So I also think the right hand side ought to be made a tiny amount lower, to cater for that.

![Łódź](https://cloud.githubusercontent.com/assets/1371085/11909179/3589edcc-a5db-11e5-8144-c090e4c98173.png)

**Comment by katef** [comment]
It might also make sense to justify the length with a slight thickening, in much the same way as the curve going towards a serif. Here's an extreme example of that:
![ldash2](https://cloud.githubusercontent.com/assets/1371085/11909619/f9536012-a5e0-11e5-9332-281c63b7f3ac.png)

**Comment by skosch** [comment]
Whoa, you're really on fire, so many issues! Thank you :)

Not a Polish speaker, so this glyph probably didn't get the love it deserves. I don't object to shortening/shift correcting the bar at all, though it is something that would have to happen in all six weights. The flaring in Galliard looks *really* neat, but you'd have to experiment with it ... all other crossbars/hyphens/dashes/etc in Crimson are straight, so it may look out of place (but I haven't tried it, so who knows).

**Comment by katef** [comment]
Specific language conventions aside, my view is that the flaring makes sense for an old style font because in theory, why would a perfectly rectangular shape ever be present? That doesn't relate to any calligraphic source I could imagine.

**Comment by katef** [comment]
Perhaps this bar should have its ends softened to match the hyphen.

**Comment by skosch** [comment]
Well, perfectly rectangular shapes do appear quite naturally when clueless teenage designers toy around with font editors :) Some of the design choices in the font were made deliberately, but when in doubt it's safe to assume they weren't.

**Comment by katef** [comment]
Here's what I'm proposing for Ł. It looks pretty crazy, but the lowering of the right hand side is to account for the optical illusion I mentioned.

I took the shape from Crimson's hyphen, and rotated it into place. This way it has the rounding at a slight slant.

I think it's slightly too light. If you're happy with the idea here, then I'll do the same for the other faces, and sort out the weight then.

![l0](https://cloud.githubusercontent.com/assets/1371085/11917988/f5495d06-a715-11e5-91d0-634d30c10c0d.png)

**Comment by skosch** [comment]
Yeah, might be ever so slightly too thin but I fully approve otherwise!

**Comment by katef** [comment]
I'm making the end caps vertical (with slightly rounded corners) instead, because that's the convention.

**Comment by katef** [comment]
Here's where I am currently. The weight is actually supposed to be around the weight of a serif's width!

![l1](https://cloud.githubusercontent.com/assets/1371085/11939454/32c04c6c-a81a-11e5-83b7-331954ecb909.png)

**Comment by katef** [comment]
I'm happy with the vertical offset to counter for the illusion of the diagonal against the stem. But I'm not happy with the way I'm trying to counter for the convex appearance along the stroke, and with the convex appearance relative to the glyphs' left edges. So I might abandon that part, and leave the edges parallel instead.

**Comment by katef** [comment]
@skosch I'm re-drawing these, and I just wanted to check you're happy with how I'm doing the ends. I couldn't find a similar curve elsewhere at this kind of angle, so I made my own. I weighted the curve slightly to the top, with inspiration from the style you have for a hyphen and the slash through Ø.

The upper node is slightly to the left of the lower one, such that the curve seems vertical on average.

![ltr0](https://cloud.githubusercontent.com/assets/1371085/11965804/17ef2d6a-a8ef-11e5-86f9-b9ef3d2073c4.png)

**Comment by skosch** [comment]
Love it! :)
